src/Compiler/RefC/RefC.idr

```diff
@@ -42,6 +42,8 @@ showcCleanStringChar '(' = ("_parenOpen" ++)
 showcCleanStringChar ')' = ("_parenClose" ++)
 showcCleanStringChar '{' = ("_braceOpen" ++)
 showcCleanStringChar '}' = ("_braceClose" ++)
+showcCleanStringChar '[' = ("_braketOpen" ++)
+showcCleanStringChar ']' = ("_braketClose" ++)
```

Comment:
This will bother me if I don't say anything even though they don't affect functionality - I guess these should be "bracket" rather than "braket"!